user-service/src/main/java/magicTheBuilder/userservice/dto/RegisterRequestDto.java

```diff
@@ -0,0 +1,24 @@
+package magicTheBuilder.userservice.dto;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class RegisterRequestDto {
+    @NotNull
+    private String username;
+    @NotNull
+    private String email;
+    @NotNull
+    private String password;
+    @NotNull
+    private String password_repeat;
```

Comment:
Pola nazywamy zgodnie z kowencują. Jeśli potrzebuujemy zmapować na coś innego to robimy to adnotacjami. Tyczy całego kodu.